CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [2.3.2](https://launchpad.net/mixxx/+milestone/2.3.3) (unreleased)
```

Comment:
2.3.3

Reply:
Do we remove the `(unreleased)` here or does that the changelog updater tool for us? I forgor :skull:

Reply:
This is a manual step during the release process. I forgot it last time. This will not happen again.

Reply:
So should the `(unreleased)` be removed here or at some point later? I'm still not familiar how our release process works to be honest.

Reply:
This will be dine during the manual gagging process.

Reply:
Ok thank you.

Reply:
I wish there was a `pre-tag` git hook that could check those things and warn if we forgot something.

Reply:
Since you "upload" tags to the remote using `git push <remote> <tagname>`, maybe you could filter for tags in the `pre-push` hook?